facebook/delphiFacebook/R/variables.R

```diff
@@ -471,3 +471,32 @@ code_vaccines <- function(input_data) {
 
   return(input_data)
 }
+
+#' Schooling
+#'
+#' @param input_data input data frame of raw survey data
+#' @return data frame augmented with `hh_number_total`
+code_schooling <- function(input_data) {
+  if ("E2_1" %in% names(input_data)) {
+    # Coded as 2 = "Yes", 3 = "No", 4 = "I don't know"
+    input_data$s_inperson_school_fulltime <- case_when(
+      input_data$E2_1 == 2 ~ 1,
+      input_data$E2_1 == 3 ~ 0,
+      TRUE ~ NA_real_
```

Comment:
Given the discussion we just had on "Don't know" answers, some thoughts. For this one, "I don't know" could be:

* school is changing rules or things are uncertain
* temporary break means they don't know what to answer
* respondent confused by the question

I think it makes sense to exclude "I don't know" here, but you could argue the other way. What's a typical rate of "I don't know" for this question?

Reply:
"I don't know" accounts for about 0.75-1% of responses.

Reply:
Ok, so I'm pretty comfortable ignoring it

Reply:
Sounds good.